torch/jit/annotations.py

```diff
@@ -106,14 +106,26 @@ def parse_type_line(type_line):
 def get_type_line(source):
     """Tries to find the line containing a comment with the type annotation."""
     lines = source.split('\n')
+    type_comment = '# type:'
+    type_lines = list(filter(lambda line: type_comment in line, lines))
```

Comment:
This will also catch a `# type:` anywhere inside the body of the function, which is not great. You can e.g. have nested functions, and the outer one will see the inner annotation!